Makefile

```diff
@@ -572,7 +571,6 @@ openstack_deploy_cleanup: namespace netconfig_deploy_cleanup ## cleans up the se
 edpm_deploy_prep: export KIND=OpenStackDataPlaneNodeSet
 edpm_deploy_prep: export EDPM_ANSIBLE_SECRET=${DATAPLANE_ANSIBLE_SECRET}
 edpm_deploy_prep: export EDPM_ANSIBLE_USER=${DATAPLANE_ANSIBLE_USER}
-edpm_deploy_prep: export EDPM_SINGLE_NODE=${DATAPLANE_SINGLE_NODE}
 edpm_deploy_prep: export EDPM_COMPUTE_IP=${DATAPLANE_COMPUTE_IP}
 edpm_deploy_prep: export EDPM_COMPUTE_1_IP=${DATAPLANE_COMPUTE_1_IP}
```

Comment:
we can drop this now

Reply:
Thanks. I fixed that in a follow up https://github.com/openstack-k8s-operators/install_yamls/pull/516